include/jwt-cpp/traits/danielaparker-jsoncons/traits.h

```diff
@@ -17,7 +17,8 @@ namespace jwt {
 		/// basic_claim's JSON trait implementation for jsoncons.
 		struct danielaparker_jsoncons {
 			// Needs at least https://github.com/danielaparker/jsoncons/commit/28c56b90ec7337f98a5b8942574590111a5e5831
-			static_assert(jsoncons::version().minor >= 167 || (jsoncons::version().major > 0), "A higher version of jsoncons is required!");
+			static_assert(jsoncons::version().minor >= 167 || (jsoncons::version().major > 0),
```

Comment:
Since you needed to upgrade the tested version, can we set the limit to 1.x?

Drop support of the 0.x release?

Reply:
done